bom/pom.xml

```diff
@@ -25,7 +25,7 @@
 
   <groupId>io.netty</groupId>
   <artifactId>netty-bom</artifactId>
-  <version>4.1.104.Final</version>
+  <version>siren-4.1.104-3-SNAPSHOT</version>
```

Comment:
```suggestion
  <version>siren-4.1.104-2-SNAPSHOT</version>
```

Reply:
The snapshot versions have been updated to `4.1.104-1` - [commit](https://github.com/sirensolutions/netty/pull/35/commits/710b39db1a98c3170ed79bc5c66f83c08501eb70)